rc4me/run.py

```diff
@@ -42,10 +32,65 @@ def cli(
         dest = Path.home()
     else:
         dest = Path(dest)
-    # Init rc4me directory variables
-    rc_dirs = RcDirs(repo, home, dest)
+
+    ctx.obj["rc_dirs"] = RcDirs(home, dest)
+
+
+@click.argument("repo", required=True, type=str)
+@cli.command()
+@click.pass_context
+def get(ctx: Dict[str, RcDirs], repo: str):
+    """Switch rc4me environment to target repo.
+
+    Replaces rc files in rc4me home directory with symlinks to files located in
+    target repo. If the target repo does not exist in the rc4me home directory,
+    the repo is cloned either locally or from GitHub.
+
+    Args:
+        repo: Target repo with rc files. May be a local repo or reference a
+            GitHub repository (e.g. jeffmm/vimrc).
+    """
+    rc_dirs = ctx.obj["rc_dirs"]
+    # Init repo variables
+    rc_dirs.set_repo(repo)
     # Clone repo to rc4me home dir or update existing local config repo
-    prepare_repo(rc_dirs, revert, reset)
+    fetch_repo(rc_dirs)
```

Comment:
Is there logging in here that makes it clear what `get` command is doing? E.g., your log message in `revert` is great.